orchestra/utils/task_lifecycle.py

```diff
@@ -16,8 +16,12 @@
 from orchestra.core.errors import TaskDependencyError
```

Comment:
might have to update tests for these code changes :)

Reply:
yep